cal/cal/templates/home_logged_in.html

```diff
@@ -26,6 +26,13 @@
         }
       });
     });
+    function runFullSync() {
+      if ("{{request.user.profile.first_time_authed}}" == "True") {
+        "{{request.user.profile.resetFirstTimeAuth}}";
+        window.location = "/v1/sync?full_sync=true&sync_all=true";
```

Comment:
I don't think you want `full_sync=true` here. If inc sync works then that is way faster than full sync. Recall that if you call inc sync and it doesn't work, then full sync will run anyway. We only want to full sync when we want a clean reset of every event.